controllers/replicationdestination.go

```diff
@@ -116,16 +117,24 @@ func (r *VolumeSnapshotRestoreReconciler) SetVSRStatus(log logr.Logger) (bool, e
 	}
 
 	if vsr.Status.Phase == volsnapmoverv1alpha1.SnapMoverRestorePhaseFailed || vsr.Status.Phase == volsnapmoverv1alpha1.SnapMoverRestorePhasePartiallyFailed {
-		return false, errors.New("vsb failed to complete")
+		return false, errors.New("vsr failed to complete")
```

Comment:
Small request: log vsr name here? Otherwise LGTM.

Reply:
Done, updated the PR!